src/components/pages/Save/index.tsx

```diff
@@ -67,21 +74,100 @@ const SaveForm: FC<{}> = () => {
   );
 };
 
-export const Save: FC<{}> = () => (
-  <SaveProvider>
-    <FormProvider formId="save">
-      <PageHeader
-        icon={<SaveIcon />}
-        title="SAVE"
-        subtitle="Earn mUSD’s native interest rate"
-      >
-        <P>
-          Deposit your mUSD into the mUSD SAVE contract and start earning
-          interest.
-        </P>
-      </PageHeader>
-      <SaveInfo />
-      <SaveForm />
-    </FormProvider>
-  </SaveProvider>
-);
+const APYStats = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-top: 2rem;
+  text-align: center;
+
+  @media (min-width: ${({ theme }) => theme.viewportWidth.xs}) {
+    margin: 0;
+    text-align: right;
+    justify-content: flex-end;
+  }
+`;
+
+const ToggleContainer = styled.div`
+  display: flex;
+`;
+
+const InfoCountUp = styled(CountUp)`
+  font-size: 1.5rem;
+  font-family: 'DM Mono', monospace;
+  color: ${({ theme }) => theme.color.blue};
+`;
+
+const InfoMsg = styled.div`
+  padding-top: 4px;
+  font-size: 12px;
+
+  a {
+    color: ${({ theme }) => theme.color.greyTransparent};
+    border: none;
+    text-decoration: underline;
+  }
+`;
+
+export const Save: FC<{}> = () => {
+  const [activeVersion, setActiveVersion] = useState(CURRENT);
+  const apyForPastWeek = useAverageApyForPastWeek();
+  const versionNumber = activeVersion === CURRENT ? 2 : 1;
+
+  const getVersion = (selection: ToggleSaveSelection): SaveVersion =>
+    selection === 'primary' ? CURRENT : DEPRECATED;
+
+  const getSelection = (version: SaveVersion): ToggleSaveSelection =>
+    version === CURRENT ? 'primary' : 'secondary';
+
+  const handleVersionToggle = useCallback(
+    (selection: ToggleSaveSelection) => setActiveVersion(getVersion(selection)),
+    [],
+  );
+
+  const handleMigrateClick = async (): Promise<void> => {
+    // await transaction trigger for withdraw.
+    // switch state and allow user to select deposit w/ new balance
+    setActiveVersion(CURRENT);
+  };
+
+  const isCurrent = activeVersion === CURRENT;
+
+  return (
+    <SaveProvider>
+      <FormProvider formId="save">
+        <PageHeader
+          title={`Save V${versionNumber}`}
+          subtitle="Earn interest on your deposited mUSD"
+        >
+          <ToggleContainer>
+            <ToggleSave
+              onClick={handleVersionToggle}
+              selection={getSelection(activeVersion)}
+            />
+          </ToggleContainer>
+          <APYStats>
+            {apyForPastWeek ? (
+              <>
+                <InfoCountUp end={apyForPastWeek} suffix="%" decimals={2} />
+                <InfoMsg>
+                  {' '}
+                  <a
+                    href="https://docs.mstable.org/mstable-assets/massets/native-interest-rate#how-is-the-24h-apy-calculated"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    Average daily APY
```

Comment:
Did we say 'over the last 7 days' previously? We should do so here for transparency.

Reply:
Can do! I removed it because it was breaking the layout on small devices – I'll give it a jiggle & get it to fit in